net_utils.js

```diff
@@ -421,3 +428,32 @@ exports.get_ipany_re = function (prefix, suffix, modifier) {
         modifier
     );
 };
+
+exports.ipv6_reverse = function(ipv6){
+    ipv6 = new IPv6.Address(ipv6);
```

Comment:
addr.toNormalizedString() would suffice for this function without the additional dependency.